app/src/main/java/com/github/sdp/mediato/api/themoviedb/models/PagedResult.java

```diff
@@ -0,0 +1,27 @@
+package com.github.sdp.mediato.api.themoviedb.models;
+
+import java.util.Iterator;
+import java.util.List;
+
+public class PagedResult<T> {
+    private int page;
```

Comment:
couldn't all those attributes be final ?

Reply:
No since they are used by the retrofit library. If we declare them final, they will need to be initialised in a public constructor. Then anyone could create an instance of the class with some data inside.

Letting them not final avoid us having TMDBMovie objects (with data) that are not provided through Retrofit.

Reply:
oh yeah sorry